tests/lib/rules/prefer-wait-for.js

```diff
@@ -0,0 +1,167 @@
+'use strict';
+
+const rule = require('../../../lib/rules/prefer-wait-for');
+const RuleTester = require('eslint').RuleTester;
+
+const ruleTester = new RuleTester({
+  parserOptions: { ecmaVersion: 2018, sourceType: 'module' },
+});
+ruleTester.run('prefer-wait-for', rule, {
+  valid: [
+    {
+      code: `async () => {
+        await waitFor(() => {});
+      }`,
+    },
+    {
+      code: `async () => {
+        await waitForElementToBeRemoved(() => {});
+      }`,
+    },
+    {
+      code: `async () => {
+        await waitForSomethingElse(() => {});
+      }`,
+    },
+    {
+      code: `import { wait } from 'some-testing-library'`,
+    },
+    {
+      code: `import { waitForElement } from 'some-testing-library'`,
+    },
+    {
+      code: `import { waitForDomChange } from 'some-testing-library'`,
```

Comment:
Can you explain what the purpose of testing only the import is?

Reply:
This was just a test I came up with to check only calls to those methods were transformed to `waitFor` but then we realized that we need to transform the imports too. I got the tests updated and suggested code from @timdeschryver almost in place. I'll push it soon for another review round.

Reply:
Ah ok, thanks!